code/modules/projectiles/ammunition/boxes.dm

```diff
@@ -110,6 +110,13 @@
 	ammo_type = /obj/item/ammo_casing/shotgun/tranquilizer
 	materials = list(MAT_METAL=1750)
 
+/obj/item/ammo_box/tranqdarts

```

Comment:
Ok, I have changed them to 9x39mm rifle rounds and updated all the icons and names to match.